multimodal/src/autogluon/multimodal/utils.py

```diff
@@ -1804,7 +1804,7 @@ def sha1sum(filename: str):
             The sha1sum
         """
         with open(filename, mode="rb") as f:
-            d = hashlib.sha1()
+            d = hashlib.new('sha1', usedforsecurity=False)  # Mark as insecure due to bandit check
```

Comment:
Is this the correct solution? This might prevent bandit to complain, but might just turned off the bandit check. Just want to double check

Reply:
Actually I decide to mark it as `nosec` because we are using sha1sum as checksums instead of using it as credentials.

Reply:
Basically, it's not protecting any sensitive contents. We can safely also use md5.